### PR TITLE
update logo

### PR DESCRIPTION
**Pull request by appleboy**
update logo file and document.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1964,14 +1964,14 @@ scss-tokenizer@^0.2.3:
...
-  version "4.3.6"
-  resolved "https://registry.yarnpkg.com/semver/-/semver-4.3.6.tgz#300bc6e0e86374f7ba61068b5b1ecd57fc6532da"
-
-semver@~5.3.0:
+"semver@2 || 3 || 4 || 5", semver@~5.3.0:
   version "5.3.0"
   resolved "https://registry.yarnpkg.com/semver/-/semver-5.3.0.tgz#9b2ce5d3de02d17c6012ad326aa6b4d0cf54f94f"
 
+semver@^4.1.0:
+  version "4.3.6"
```
Comment (bkcsoft) [inline yarn.lock:1972]: Why is there 2 `semver` versions?

**Review by bkcsoft — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (appleboy) [reply yarn.lock:1972]: I just execute `yarn install` so this file has been updated automatically.

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 				<div class="row">
 					<div class="col-xs-12 text-xs-center">
 						<p>
-							Copyright &copy; {{ .Now.Year }} <a href="{{ .Site.Params.website }}">{{ .Site.Params.author }}</a>. All rights reserved. Made with <i class="fa fa-heart" aria-hidden="true"></i> and <a href="https://gohugo.io">Hugo</a>. Sponsored by <a href="http://inblockchain.com">INBlockchain</a>.
+							Copyright &copy; {{ now.Year }} <a href="{{ .Site.Params.website }}">{{ .Site.Params.author }}</a>. All rights reserved. Made with <i class="fa fa-heart" aria-hidden="true"></i> and <a href="https://gohugo.io">Hugo</a>. Sponsored by <a href="http://inblockchain.com">INBlockchain</a>.
```
Comment (appleboy) [inline layouts/partials/footer.html:6]: fix `ERROR 2017/08/02 07:25:20 Page's Now is deprecated and will be removed in Hugo 0.26. Use now (the template func).` error from hugo.

**Comment by lunny** [comment]
LGTM

File: yarn.lock (hunk 1 above)
Comment (lafriks) [reply yarn.lock:1972]: @bkcsoft that is because gulp requires one version and gulp-sass depenency requires other version

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Pull request merged**